src/components/IntoleranceFilterTable.js

```diff
@@ -0,0 +1,186 @@
+import styled from "styled-components";
+
+export default function IntoleranceFilterInformation() {
+  return (
+    <StyledTable>
```

Comment:
I'm wondering whether there was a more automated way of generating the table content, like fetching for all possible intolerances?

Reply:
Would've been great, but unfortunately the API doesn't provide the definitions for the intolerances - I had to copy & paste them from the documentation...